modules/_labs/breadcrumbs/react/lib/Breadcrumbs/Dropdown/Button.tsx

```diff
@@ -0,0 +1,42 @@
+/** @jsx jsx */
+import React from 'react';
+import {css, jsx} from '@emotion/core';
+import {IconButton, IconButtonProps} from '@workday/canvas-kit-react-button';
+import {folderCloseIcon} from '@workday/canvas-system-icons-web';
+import {CanvasSystemIcon} from '@workday/design-assets-types';
+import {colors} from '@workday/canvas-kit-react-core';
+
+export interface DropdownButtonProps extends IconButtonProps {
+  'aria-label': string;
+  buttonIcon?: CanvasSystemIcon;
+  buttonRef: React.Ref<HTMLButtonElement>;
+  toggled: boolean;
+}
+
+export const DropdownButton = ({
+  buttonIcon = folderCloseIcon,
+  buttonRef,
+  toggled,
+  variant = IconButton.Variant.Plain,
+  ...props
+}: DropdownButtonProps) => {
+  const hasPlainVariant = variant === IconButton.Variant.Plain;
+  // gives the plain icon variant a little extra space between the chevron separators
+  const customStyles = css(hasPlainVariant ? {margin: '0 1px'} : {});
+  return (
+    <IconButton
+      type="button"
+      css={customStyles}
+      variant={variant}
+      icon={buttonIcon}
+      color={colors.licorice200}
+      buttonRef={buttonRef}
+      toggled={toggled}
+      aria-pressed={undefined}
```

Comment:
Any reason this needs to be `undefined`, I assume it's to override what `IconButton` naturally does when `toggled` is defined but this seems mysterious to me without knowing the context.

Reply:
Good question. Yes, that's exactly it. We're using `aria-expanded` here instead of `aria-pressed`, and `undefined` removes the prop. We're doing that in the [Pagination component](https://github.com/Workday/canvas-kit/blob/master/modules/_labs/pagination/react/lib/Pages.tsx#L78) as well. But I agree the intent is unclear. I'll add a comment.